### ISSUE TITLE
Skill don't work

### ISSUE THREAD

**Issue by Bettiol** [ndg63276/alexa-youtube#30]
Hello, from today the skill no longer works. He always tells me:
```

Skill response was marked as failure
My YouTube Skill
Request Identifier: amzn1.echo-api.request.---------------------
```

The target Lambda application returned a failure response

**Comment by ndg63276** [comment]
*sigh*

Could you supply any more information? Anything at all? How am I supposed to debug what you've reported? You went to the effort to disguise your request identifier, as if that is your credit card number or something, why not add some useful information?

1. What are you saying to Alexa?
2. What video are you trying to play? (URL)
3. Are you using your own lambda instance or mine?
4. When did you last update?
5. Does 'Alexa, ask YouTube to play Gangnam Style' work?

**Comment by Bettiol** [comment]
> _sigh_
> 
> Could you supply any more information? Anything at all? How am I supposed to debug what you've reported? You went to the effort to disguise your request identifier, as if that is your credit card number or something, why not add some useful information?
> 
> 1. What are you saying to Alexa?
> 2. What video are you trying to play? (URL)
> 3. Are you using your own lambda instance or mine?
> 4. When did you last update?
> 5. Does 'Alexa, ask YouTube to play Gangnam Style' work?

Hi. Sorry I didn't provide any useful information right away.
So, I installed the skill on January 6, 2019 directly using my lambda instance. Until 2 days ago, I had never had any problems.
Now it no longer works, or rather sometimes it works badly, because it plays videos that make no sense with what I told him. 95% of the time, however, has always been the problem:
Skill response was marked as failure
My YouTube Skill
Request Identifier: amzn1.echo-api.request.--------------------------------------------------------------------
The Lambda target application returned a failure response

I've tried all the phrases you put on README.md, but they all give the above error or play back videos that don't match anything with what I asked them to do.

https://youtu.be/MNWpYoR0HLw

I'm using my instance lambda

I installed the skill on January 6, so I installed the last update of the 4.

"Alexa, ask YouTube to play Gangnam Style" does not work.
